TESTS/network/emac/main.cpp

```diff
@@ -72,7 +72,10 @@ Case cases[] = {
     Case("EMAC unicast frame length", test_emac_unicast_frame_len),
     Case("EMAC unicast burst", test_emac_unicast_burst),
     Case("EMAC unicast long", test_emac_unicast_long),
+#if !defined(TARGET_UBLOX_EVK_ODIN_W2) && \
+    !defined(TARGET_MTB_UBLOX_ODIN_W2)
```

Comment:
This should be platform agnostic code.
Can we remove ifdefs that involve specific platforms and instead use label/features/macro that could be enabled in targets.json?
For example SUPPORT_EMAC_MULTICAST
FYI @bulislaw

Reply:
Implemented in correct way.